composer.lock

```diff
@@ -0,0 +1,2107 @@
+{
```

Comment:
please remove this file and add this to .gitignore  - there is no need to block versions in the library - so there is no point in commenting composer.lock